src/main/java/com/example/demo/config/MailConfig.java

```diff
@@ -0,0 +1,51 @@
+package com.example.demo.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+
+import java.util.Properties;
+
+@Configuration
+public class MailConfig {
```

Comment:
It's good that you found how to configure mail sender yourself, but you don't need to do it when using starter. It preconfigure this for you based on `application.properties`

Reply:
Yes, you are right